packages/bot-web-ui/src/constants/contract.js

```diff
@@ -0,0 +1,225 @@
+    STOP_LOSS: localize('Stop loss:'),
+    TOTAL_PROFIT_LOSS: localize('Total profit/loss:'),
+    PROFIT_LOSS: localize('Profit/Loss:'),
+    POTENTIAL_PROFIT_LOSS: localize('Potential profit/loss:'),
+    INDICATIVE_PRICE: localize('Indicative price:'),
+    PAYOUT: localize('Payout:'),
+    PURCHASE_PRICE: localize('Purchase price:'),
+    POTENTIAL_PAYOUT: localize('Potential payout:'),
+    TICK: localize('Tick '),
+    WON: localize('Won'),
+    LOST: localize('Lost'),
+    DAYS: localize('Days'),
+    DAY: localize('Day'),
+    SELL: localize('Sell'),
+});
+
+export const getMarketNamesMap = () => ({
+    FRXAUDCAD: localize('AUD/CAD'),
+    FRXAUDCHF: localize('AUD/CHF'),
+    FRXAUDJPY: localize('AUD/JPY'),
+    FRXAUDNZD: localize('AUD/NZD'),
+    FRXAUDPLN: localize('AUD/PLN'),
+    FRXAUDUSD: localize('AUD/USD'),
+    FRXBROUSD: localize('Oil/USD'),
+    FRXEURAUD: localize('EUR/AUD'),
+    FRXEURCAD: localize('EUR/CAD'),
+    FRXEURCHF: localize('EUR/CHF'),
+    FRXEURGBP: localize('EUR/GBP'),
+    FRXEURJPY: localize('EUR/JPY'),
+    FRXEURNZD: localize('EUR/NZD'),
+    FRXEURUSD: localize('EUR/USD'),
+    FRXGBPAUD: localize('GBP/AUD'),
+    FRXGBPCAD: localize('GBP/CAD'),
+    FRXGBPCHF: localize('GBP/CHF'),
+    FRXGBPJPY: localize('GBP/JPY'),
+    FRXGBPNOK: localize('GBP/NOK'),
+    FRXGBPUSD: localize('GBP/USD'),
+    FRXNZDJPY: localize('NZD/JPY'),
+    FRXNZDUSD: localize('NZD/USD'),
+    FRXUSDCAD: localize('USD/CAD'),
+    FRXUSDCHF: localize('USD/CHF'),
+    FRXUSDJPY: localize('USD/JPY'),
+    FRXUSDNOK: localize('USD/NOK'),
+    FRXUSDPLN: localize('USD/PLN'),
+    FRXUSDSEK: localize('USD/SEK'),
+    FRXXAGUSD: localize('Silver/USD'),
+    FRXXAUUSD: localize('Gold/USD'),
+    FRXXPDUSD: localize('Palladium/USD'),
+    FRXXPTUSD: localize('Platinum/USD'),
+    OTC_AEX: localize('Dutch Index'),
+    OTC_AS51: localize('Australian Index'),
+    OTC_DJI: localize('Wall Street Index'),
+    OTC_FCHI: localize('French Index'),
+    OTC_FTSE: localize('UK Index'),
+    OTC_GDAXI: localize('German Index'),
+    OTC_HSI: localize('Hong Kong Index'),
+    OTC_IBEX35: localize('Spanish Index'),
+    OTC_N225: localize('Japanese Index'),
+    OTC_NDX: localize('US Tech Index'),
+    OTC_SPC: localize('US Index'),
+    OTC_SSMI: localize('Swiss Index'),
+    OTC_SX5E: localize('Euro 50 Index'),
+    R_10: localize('Volatility 10 Index'),
+    R_25: localize('Volatility 25 Index'),
+    R_50: localize('Volatility 50 Index'),
+    R_75: localize('Volatility 75 Index'),
+    R_100: localize('Volatility 100 Index'),
+    RDBEAR: localize('Bear Market Index'),
+    RDBULL: localize('Bull Market Index'),
+    WLDAUD: localize('AUD Index'),
+    WLDEUR: localize('EUR Index'),
+    WLDGBP: localize('GBP Index'),
+    WLDUSD: localize('USD Index'),
+    '1HZ10V': localize('Volatility 10 (1s) Index'),
+    '1HZ100V': localize('Volatility 100 (1s) Index'),
+});
+
+export const getUnsupportedContracts = () => ({
```

Comment:
We support all contracts in DBot, not sure whether this function is required, or its values should be concatenated to `getSupportedContracts`.